packages/lesswrong/components/tagging/TagEditsTimeBlock.tsx

```diff
@@ -38,6 +38,19 @@ const TagEditsTimeBlock = ({before, after, reportEmpty, classes}: {
         users {
           ...UsersMinimumInfo
         }
+        documentDeletions {
```

Comment:
This can wait for a followup later, but ideally this would be a bit more general; ie we don't only want to show deletions, we also want to be able to show renames and advanced-metadata changes.